src/ir/module-splitting.cpp

```diff
@@ -542,7 +544,7 @@ void ModuleSplitter::setupTablePatching() {
       placeholder->base = std::to_string(index);
       placeholder->name = Names::getValidFunctionName(
         primary, std::string("placeholder_") + placeholder->base.toString());
-      placeholder->hasExplicitName = false;
+      placeholder->hasExplicitName = true;
```

Comment:
I'm not sure about this change. Alternatively, we can keep this to false and change a few tests.

Reply:
@tlively any thoughts here?

Reply:
The thing is that there are a number of tests with an output like:
```
(import "placeholder" "0" (func $placeholder_0 (param i32) (result i32)))
```
If I keep it to `false`, the internal function name is no longer as explicit.

Reply:
I would be fine keeping this `false` and updating the tests.